style.css

```
@@ -39,3 +77,98 @@ p {
   box-shadow: 0 0 25px rgba(0, 0, 0, 0.25);
   background: white;
 }
+
+/* 대기중 개수 */
+.wait {
+  padding: 15px 15px 5px 15px;
+  font-weight: bold;
+  font-size: 17px;
+}
+
+/* 완료 개수 */
+.complete {
+  padding: 15px 15px 5px 15px;
+  font-weight: bold;
+  font-size: 17px;
+}
+
+/* 완료 시 선긋기 */
+#completeList {
+  text-decoration: line-through;
+  color: #a7a7a7;
+}
+
+/* todo 목록 */
+.list {
+  height: 10px;     /* list div의 크기 고정 위해 */
+  flex-grow: 8;
+  padding: 10px 10px;
+
+  overflow: scroll;
+  overflow-x: hidden;
+
+  /* inner items */
+  display: flex;
+  flex-direction: column;
+}
+
+/* 할 일 추가 */
+.add {
+  display: inline-flex;
+  flex-grow: 0.8;
+
+  justify-content: center;
+  align-items: center;
+
+  padding: 0 10px;
+  box-shadow: 0 0 5px rgba(0, 0, 0, 0.25);
+}
+
+/* 할 일 추가 input */
+#add_input {
+  /* input */
+  width: 100%;
+  height: 70%;
+  border-width: 0.7px;
+  border-radius: 20px;
+  border-color: #c4c4c4;
+  outline: none;
+
+  align-items: stretch;
+
+  /* text */
+  text-align: center;
+  font-size: 15px;
+}
+
+#add_input::placeholder {
+  color: #c4c4c4;
+}
+
+/* trashcan image */
+.listbox img {
+  width: 20px;
+  height: 20px;
+  
+  opacity: 0;
+  transition: 0.3s;
+}
+
+.listbox img:hover {
+  opacity: 1;
+}
```

UI 상으로 `.listbox img:hover`보다는 `.listbox:hover img`가 더 좋을 것 같아요.
처음 사용해보는 사람들이 지우는 기능을 알려면 trash bin이 있는 이미지 쪽으로 마우스를 올려야 하니까, listbox에 올려도 지우는 기능이 있다는 걸 보여주는게 좋을듯?
```suggestion
.listbox img {
  width: 20px;
  height: 20px;
  
  opacity: 0;
  transition: 0.3s;
}

.listbox:hover img {
  opacity: 1;
}
```